web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/views/filter/ViewsBasicFilter.php

```diff
@@ -1,40 +0,0 @@
-<?php
-
-namespace Drupal\ys_views_basic\Plugin\views\filter;
```

Comment:
Quick question, is this file Deletion ok? , since this PR is to implement a pager I don't see a reason to remove a existing filter here

Reply:
Ahh yes, this is correct. I moved the custom filter into contextual filters so it's easier to support multiple content types and tags. Those are future features - for the moment, only one content type and one tag can be applied.